problem.py

```diff
@@ -199,16 +218,14 @@ def _step_detection_recall(step_list_true, step_list_pred):
     Returns:
         float -- recall, between 0.0 and 1.0
     """
-    _check_step_list(step_list_true)
     _check_step_list(step_list_pred)
 
     n_detected_true = 0
     predicted_index_set = set()  # set of indexes of predicted steps
```

Comment:
ditto.

Reply:
solved in d1e73021a2b584f44b2fbff2c05bbdeb6b39c312